include/z64.h

```diff
@@ -1643,6 +1643,17 @@ typedef struct ArenaNode {
     /* 0x28 */ u8 unk_28[0x30-0x28]; // probably padding
 } ArenaNode; // size = 0x30
 
+#define RELOC_SECTION(reloc) ((reloc) >> 30)
+#define RELOC_OFFSET(reloc) ((reloc) & 0xFFFFFF)
+#define RELOC_TYPE(reloc) ((reloc) & 0x3F000000)
+
+typedef enum OverlayRelocationType {
+    R_MIPS_32 = 2,
+    R_MIPS_26 = 4,
+    R_MIPS_HI16 = 5,
+    R_MIPS_LO16 = 6
+} OverlayRelocationType;
```

Comment:
These are more generally MIPS relocation types, you can find them elsewhere such as ELF files for any MIPS machine. It's somewhat confusing to call the "Overlay" relocation types since it implies that they're something specific to the overlay implementation

Reply:
True, for example `mips-linux-gnu-objdump --disassemble --reloc build/src/overlays/actors/ovl_En_Daiku/z_en_daiku.o` (excerpt)
```
    1488:       3c040000        lui     a0,0x0
                        1488: R_MIPS_HI16       .data
    148c:       15e10012        bne     t7,at,14d8 <EnDaiku_PostLimbDraw+0x8c>
    1490:       8fa50048        lw      a1,72(sp)
    1494:       24840124        addiu   a0,a0,292
                        1494: R_MIPS_LO16       .data
    1498:       0c000000        jal     0 <EnDaiku_ChangeAnim>
                        1498: R_MIPS_26 Matrix_MultVec3f
    149c:       24a50038        addiu   a1,a1,56
```

Reply:
Makes sense. Changed it to `MIPSRelocationType`.